faunadb-common/src/main/java/com/faunadb/common/Connection.java

```diff
@@ -149,6 +150,18 @@ public Builder withJvmDriver(JvmDriver jvmDriver) {
       return this;
     }
 
+    /**
+     * Sets the JVM version to use for the connection.
+     *
+     * @param jvmVersion
+     * @return this {@link Builder} object
+     */
+    public Builder withJvmVersion(String jvmVersion) {
```

Comment:
This name is a bit confusing. It's value isn't the JVM version, right? It looks like the language version. Is it?